daemon/algod/api/algod.oas2.json

```diff
@@ -1246,6 +1246,130 @@
         }
```

Comment:
Are you referring to the sig-type of the StateProof transaction? I suppose it could be nil, from what I see it's mostly used by the indexer.
How is the sig-type handled for other special accounts such as Rewards Pool or Fee Sink?

Reply:
@winder, could you please help with this?

Reply:
The fee sink and rewards pool don't send transactions. AFAIK they are used as part of the consensus protocol.

I don't know how this should be handled. What is your suggestion?

Reply:
I believe the fee sink _could_ send transactions, but it is constrained by the protocol to only send to the rewards pool.  Therefore, I _think_ the foundation has the private key associated with it, but has not used it.

None of this helps answer the question at hand.

Reply:
On test networks, it is very easy to assign the fee sink and rewards pool addresses in the genesis file to an account you have the private key for..

Reply:
There shouldn't be any private key for signing stateproof transactions, I think it's a good assurance to use an address which we couldn't possibly have the private key for, to inspire trust in the mechanism.

Circling back to the original question, since the stateproof transaction is not actually signed but verified cryptographically instead, I believe the `sig-type` parameter is not relevant to it.
I don't see any reason (in the code) why this couldn't be `nil`, does that seems fine to you?